core/src/main/java/org/apache/iceberg/jdbc/JdbcUtil.java

```diff
@@ -73,6 +99,8 @@ final class JdbcUtil {
           + " VARCHAR(1000),"
           + JdbcTableOperations.PREVIOUS_METADATA_LOCATION_PROP
           + " VARCHAR(1000),"
+          + TYPE
+          + " VARCHAR(5),"
```

Comment:
I wonder if we should make this slightly larger than 5 (just to make it future-proof)